nautobot/docs/additional-features/jobs.md

```diff
@@ -814,13 +819,13 @@ class NewBranch(Job):
         }
     )
 
-    def run(self, data, commit):
+    def run(self, site_name, switch_count, switch_model):
```

Comment:
Example needs to be modified to use Location rather than Site throughout - not blocking.